### DISCUSSION TITLE
Is there a Docker Compose file I can copy and use?

### DISCUSSION THREAD

**Discussion by mrtargaryen** [open-nomie/nomie6-oss#81] [Q&A] [answered]
I'm really interested in this app and want to use it in Docker.

**Comment by ZL1LAC** [comment, marked as answer]
Someone has made compose file with CouchDB support in #11 that you can try.

**Reply by mrtargaryen** [reply]
Got it working, thank you!

**Comment by ZL1LAC** [comment]
Your welcome, happy to help.
